gpflux/layers/basis_functions/fourier_features/random.py

```diff
@@ -122,6 +123,43 @@ def _compute_constant(self) -> tf.Tensor:
         return self.rff_constant(self.kernel.variance, output_dim=2 * self.n_components)
 
 
+class QuasiRandomFourierFeatures(RandomFourierFeatures):
+
+    r"""
+    Quasi-random Fourier features (ORF) :cite:p:`yang2014quasi` for more
```

Comment:
Looks like the acronym is wrong?
```suggestion
    Quasi-random Fourier features (???) :cite:p:`yang2014quasi` for more
```